deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -51,6 +51,8 @@ monitoring:
   # -- Enable Prometheus integration, will also create necessary RBAC rules to allow Operator to create ServiceMonitors.
   # Monitoring requires Prometheus to be pre-installed
   enabled: false
+  # -- Whether to disable the metrics reported by Ceph. If false, the prometheus mgr module and Ceph exporter are enabled
+  metricsDisabled: true
```

> I see https://github.com/rook/rook/blob/master/Documentation/CRDs/Cluster/ceph-cluster-crd.md it's default value is false.
> 
> metricsDisabled: Whether to disable the metrics reported by Ceph. If false, the prometheus mgr module and Ceph exporter are enabled. If true, the prometheus mgr module and Ceph exporter are both disabled. Default is false.

Let's leave the default as false. Other users can change it to true when desired.
```suggestion
  metricsDisabled: false
```

---

Done